tests_integ/helpers/deployer/utils/table_print.py

```diff
@@ -0,0 +1,137 @@
+"""
+Utilities for table pretty printing using click
+"""
+from itertools import count
+
+try:
+    from itertools import zip_longest
+except ImportError:  # py2
+    from itertools import izip_longest as zip_longest
+import textwrap
+from functools import wraps
+
+import click
+
+MIN_OFFSET = 20
+
+
+def pprint_column_names(format_string, format_kwargs, margin=None, table_header=None, color="yellow"):
```

Comment:
What does this do?

I assume it's to improve legibility? Do we expect changes to this? It's worth considering the long-term maintenance impact.